wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/templates/sample/Robot.java

```diff
@@ -91,24 +91,31 @@ public void robotInit() {
 	 */
 	@Override
 	public void autonomous() {
-		String autoSelected = m_chooser.getSelected();
-		// String autoSelected = SmartDashboard.getString("Auto Selector",
-		// defaultAuto);
+		String autoSelected = SmartDashboard.getString("Auto Selector",
+				kDefaultAuto);
 		System.out.println("Auto selected: " + autoSelected);
 
+		// Warning: while disabling motor safety is useful for simple examples,
+		// don't use it in production robot code.
+		m_robotDrive.setSafetyEnabled(false);
+
 		switch (autoSelected) {
 			case kCustomAuto:
-				m_robotDrive.setSafetyEnabled(false);
-				m_robotDrive.arcadeDrive(-0.5, 1.0); // spin at half speed
-				Timer.delay(2.0); // for 2 seconds
-				m_robotDrive.arcadeDrive(0.0, 0.0); // stop robot
+				// Spin at half speed for two seconds
+				m_robotDrive.arcadeDrive(0.0, 1.0);
```

Comment:
This will spin at full speed.

Reply:
#768 is fixing this. I don't want to step on that PR's toes.

Reply:
Yeah I sense a small merge conflict.